pset-iterators.js

```diff
@@ -43,6 +43,11 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const firstWhigPresident = presidents.find((president) => {
+  return president.party === 'Whig'
+}).president
```

Comment:
I love that you traversed through the returned object at the end here; brilliant.